### ISSUE TITLE
Reload clojure-lsp settings without reloading the vscode window

### ISSUE THREAD

**Issue by sauvala** [BetterThanTomorrow/calva#1113]
Changing the clojure-lsp settings in `./lsp/config.edn` requires reloading the vscode window. It would be cool if there would a way to reload clojure-lsp without reloading vscode.

**Comment by ericdallo** [comment]
we could make clojure-lsp always read again from `.lsp/config.edn`, not sure if this would affect performance as we check settings all the time for multiple features, as the last resource maybe use a small cache of some seconds?

Suggestinos @snoe @borkdude?

**Comment by PEZ** [comment]
I saw some new library flash by somewhere, extracting efficient file watching out of Krell. But my experience from Calva to read the config every time is that it is cheap. Iirc, Calva reads the formatter config on each invocation, which is often.

**Comment by ericdallo** [comment]
Yeah, but I bet VSCode has some kind of cache for those settings, for clojure-lsp case, we would need to read a file from disk
